### ISSUE TITLE
react/prop-types false positive on component class with static get propTypes()

### ISSUE THREAD

**Issue by hugihlynsson** [yannickcr/eslint-plugin-react#48]
If I have a class that has a property called props, the react/prop-types rule will complain about missing props validation even if it is defined in a `static get propTypes`
```js
import React from 'react';
class MyClass extends React.Component {
  static get propTypes() {
    return {
      test: React.PropTypes.bool,
    };
  }
  myMethod() {
    return this.props.test;
  }
}
```
`9:11 error 'test' is missing in props validation for MyClass react/prop-types`
This should not happen since the test prop is being validated

eslint-plugin-react 2.0.2

**Comment by yannickcr** [comment]
Duplicate of #43, statics are not supported (yet).

**Comment by yannickcr** [comment]
Sorry, it was not exactly the same issue since this one is about static methods and the other one is about static properties.
